### PR TITLE
feat(launcher): Enable specification of retry-limit

### LINKED ISSUE #1126: Allow configuration for retryLimit
It would be really helpful to allow a config param in `karma.conf.js` for retryLimit for browser crashes.

It would avoid us to wrap the `grunt` or `karma` call in a command like:
```bash
$ for i in `seq 1 3`; do grunt karma:ci; test $? -ne 3 && break; done
```
command.

I don't see any available configuration in http://karma-runner.github.io/0.12/config/configuration-file.html, but I do see it in the code a possibility to pass it: https://github.com/karma-runner/karma/blob/9e0e3854853dc04dd273dd0c17878d76de24fb70/lib/launchers/retry.js#L3

Could we make this setting available in config file?
Thanks!

### PR DESCRIPTION
**Pull request by budde377**
Allow specification of retry limit in configuration file with the `retryLimit` property. 
I also reordered the entries in the documentation in alphabetical order.

Closes #1126

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -265,6 +265,7 @@ var Config = function () {
   this.browserNoActivityTimeout = 10000
   this.concurrency = Infinity
   this.failOnEmptyTestSuite = true
+  this.retryLimit = 1
```
Comment (Dignifiedquire) [inline lib/config.js]: This should be `2` right?

Comment (budde377) [inline lib/config.js]: Yes

**Comment by Dignifiedquire** [comment]
This looks good to me, could you squash the last commit into the first please?

**Comment by budde377** [comment]
Done

**Comment by Dignifiedquire** [comment]
Thanks :octocat:

**Pull request merged**